homeassistant/components/smarttub/const.py

```diff
@@ -25,3 +25,5 @@
 ATTR_PUMPS = "pumps"
 ATTR_REMINDERS = "reminders"
 ATTR_STATUS = "status"
+
+CONF_CONFIG_ENTRY = "config_entry"
```

Comment:
This doesn't seem used.

Reply:
Addressed in https://github.com/home-assistant/core/pull/49579